docker/docker-compose.yml

```diff
@@ -27,5 +26,5 @@ services:
     cap_add:
       - NET_ADMIN
     user: build
-    command: /bin/bash
+    command: /bin/sh -c "sudo mount binfmt_misc -t binfmt_misc /proc/sys/fs/binfmt_misc && /bin/bash"
```

Comment:
Why should we use "command" to mount the binfmt_misc instead of "volumes"?

Reply:
Because you can't mount specific directory under the  /proc.

```
$ docker run -it --rm -v /proc/sys/fs/binfmt_misc:/proc/sys/fs/binfmt_misc debian:12 ls
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/proc/sys/fs/binfmt_misc" to rootfs at "/proc/sys/fs/binfmt_misc": "/var/lib/docker/overlay2/f044fcb8cdb9c28329a1674752db06ea344648cf26a8ee1431620ad32a4157bb/merged/proc/sys/fs/binfmt_misc" cannot be mounted because it is inside /proc: unknown.
```